index.html

```diff
@@ -795,18 +795,20 @@ <h2>
         The user agent MUST navigate the application context as per [[!HTML]]'s
         <a>navigate algorithm</a> with exceptions enabled. If the URL of the
         resource being loaded in the navigation is not <a>within scope</a> of
-        the navigation scope, then the user agent MUST behave as if the
-        application context is not <a>allowed to navigate</a> (this provides
-        the ability for the user agent to perform the navigation in a different
-        browsing context, or in a different user agent entirely). If during the
-        handle redirects step of HTML's <a>navigate algorithm</a> the redirect
-        URL is not <a>within scope</a>, abort HTML's navigation algorithm with
-        a <code>SecurityError</code>.
+        the navigation scope of the application context's manifest, then the
+        user agent MUST behave as if the application context is not <a>allowed
+        to navigate</a> (this provides the ability for the user agent to
+        perform the navigation in a different browsing context, or in a
+        different user agent entirely). If during the handle redirects step of
```

Comment:
Can we link to 'handle redirects step'  definition

Reply:
I can't find it anywhere. The navigate algorithm doesn't seem to deal with redirects at all; that is taken care of by the HTTP fetch algorithm (in the fetch spec).

Looks like things have been refactored. I think this change is out of scope from this PR, especially since I plan to rework things shortly in this area.